src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,39 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
```

Comment:
Consider whether `storageDao` should be initialized in a `@BeforeEach` or `@BeforeAll` annotated method. This choice depends on whether you want a fresh instance before each test (`@BeforeEach`) or one shared instance for all tests (`@BeforeAll`).